{"timestamp":"2025-07-17T05:19:17.663629","level":"info","event":"DAG bundles loaded: dags-folder, example_dags","logger":"airflow.dag_processing.bundles.manager.DagBundlesManager"}
{"timestamp":"2025-07-17T05:19:17.665407","level":"info","event":"Filling up the DagBag from /opt/airflow/dags/test.py","logger":"airflow.models.dagbag.DagBag"}
{"timestamp":"2025-07-17T05:19:17.739653Z","level":"info","event":"Task instance is in running state","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-17T05:19:17.740423Z","level":"info","event":" Previous state of the Task instance: TaskInstanceState.QUEUED","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-17T05:19:17.741117Z","level":"info","event":"Current task name:run_notebook","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-17T05:19:17.742448Z","level":"info","event":"Dag name:databricks_notebook_trigger2","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-17T05:19:17.755869","level":"info","event":"Connection Retrieved 'databricks_default'","logger":"airflow.hooks.base"}
{"timestamp":"2025-07-17T05:19:17.760054","level":"info","event":"Using token auth. For security reasons, please set token in Password field instead of extra","logger":"airflow.task.hooks.airflow.providers.databricks.hooks.databricks.DatabricksHook"}
{"timestamp":"2025-07-17T05:19:26.029178","level":"error","event":"Attempt 1 API Request to Databricks failed with reason: <Future at 0x7ff7e1ea3e90 state=finished raised ConnectionError>","logger":"airflow.task.hooks.airflow.providers.databricks.hooks.databricks.DatabricksHook"}
{"timestamp":"2025-07-17T05:19:28.012487","level":"error","event":"Task failed with exception","logger":"task","error_detail":[{"exc_type":"AirflowException","exc_value":"Response: {\"error_code\":\"INVALID_PARAMETER_VALUE\",\"message\":\"Only serverless compute is supported in the workspace.\",\"details\":[{\"@type\":\"type.googleapis.com/google.rpc.RequestInfo\",\"request_id\":\"89e4ff8b-94bf-4709-9b83-59a02f0aea46\",\"serving_data\":\"\"}]}, Status Code: 400","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":877,"name":"run"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1164,"name":"_execute_task"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/operator.py","lineno":397,"name":"wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/databricks/operators/databricks.py","lineno":641,"name":"execute"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/databricks/hooks/databricks.py","lineno":328,"name":"submit_run"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/databricks/hooks/databricks_base.py","lineno":698,"name":"_do_api_call"}],"is_group":false,"exceptions":[]},{"exc_type":"HTTPError","exc_value":"400 Client Error: Bad Request for url: https://dbc-c197be75-7f15.cloud.databricks.com/api/2.1/jobs/runs/submit","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/databricks/hooks/databricks_base.py","lineno":672,"name":"_do_api_call"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/tenacity/__init__.py","lineno":445,"name":"__iter__"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/tenacity/__init__.py","lineno":378,"name":"iter"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/tenacity/__init__.py","lineno":400,"name":"<lambda>"},{"filename":"/usr/local/lib/python3.12/concurrent/futures/_base.py","lineno":449,"name":"result"},{"filename":"/usr/local/lib/python3.12/concurrent/futures/_base.py","lineno":401,"name":"__get_result"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/databricks/hooks/databricks_base.py","lineno":691,"name":"_do_api_call"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/requests/models.py","lineno":1026,"name":"raise_for_status"}],"is_group":false,"exceptions":[]}]}
{"timestamp":"2025-07-17T05:19:28.131754","level":"warning","event":"No XCom value found; defaulting to None.","key":"run_page_url","dag_id":"databricks_notebook_trigger2","task_id":"run_notebook","run_id":"manual__2025-07-17T05:19:15.019416+00:00","map_index":-1,"logger":"task"}
{"timestamp":"2025-07-17T05:19:28.142306","level":"error","event":"Top level error","logger":"task","error_detail":[{"exc_type":"AirflowRuntimeError","exc_value":"API_SERVER_ERROR: {'status_code': 422, 'message': 'Remote server returned validation error', 'detail': {'detail': [{'type': 'missing', 'loc': ['body'], 'msg': 'Field required', 'input': None}]}}","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1302,"name":"main"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1242,"name":"finalize"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":546,"name":"_xcom_push_to_db"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/xcom.py","lineno":116,"name":"_set_xcom_in_db"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/comms.py","lineno":203,"name":"send"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/comms.py","lineno":263,"name":"_get_response"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/comms.py","lineno":250,"name":"_from_frame"}],"is_group":false,"exceptions":[]}]}
{"timestamp":"2025-07-17T05:19:28.166762Z","level":"warning","event":"Process exited abnormally","exit_code":1,"logger":"task"}
